setup.py

```diff
@@ -165,14 +152,23 @@ def get_install_requires():
         ]
     },
     package_data={'pytype': get_data_files()},
-    install_requires=get_install_requires(),
+    python_requires='>=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*, <3.8',
+    install_requires=[
+        'attrs',
+        'importlab>=0.5.1',
+        'ninja',
+        'pyyaml>=3.11',
+        'six',
+        'typed_ast; python_version >= "3.3"',
+    ],
     classifiers=[
         'Development Status :: 5 - Production/Stable',
         'Intended Audience :: Developers',
         'License :: OSI Approved :: Apache Software License',
         'Programming Language :: Python',
         'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python :: 3.7',
```

Comment:
Could you add 3.5 to this list? I'm pretty sure we just forgot to update it when we decided to support 3.5.

Reply:
Done (and ported this change to #509).